chart/helm-operator/dashboards/helm-operator-dashboard.json

```diff
@@ -0,0 +1,703 @@
+{
+  "__inputs": [
+    {
+      "name": "DS_PROMETHEUS",
+      "label": "Prometheus",
+      "description": "",
+      "type": "datasource",
+      "pluginId": "prometheus",
+      "pluginName": "Prometheus"
+    },
+    {
+      "name": "DS_LOKI",
+      "label": "Loki",
+      "description": "",
+      "type": "datasource",
+      "pluginId": "loki",
+      "pluginName": "Loki"
+    }
+  ],
+  "__requires": [
+    {
+      "type": "grafana",
+      "id": "grafana",
+      "name": "Grafana",
+      "version": "7.0.5"
+    },
+    {
+      "type": "panel",
+      "id": "graph",
+      "name": "Graph",
+      "version": ""
+    },
+    {
+      "type": "panel",
+      "id": "logs",
+      "name": "Logs",
+      "version": ""
+    },
+    {
+      "type": "datasource",
+      "id": "loki",
```

Comment:
One suggestion, it would be nice to have the usage of Loki customizable. Although it's a great tool, for any other constraint there would be projects that are not using Loki. A flag that would enable/disable the logs in the dashboard would be awesome.

Reply:
I've seen some dashboards in other OSS with Loki, it's the default way to see logs in Grafana. 

That being said, it would be nice to have this toggle, but in the end, the dashboard will still be working (with or without loki)